database_sqlite.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"database/sql"
+	"log"
+	"os"
+	"time"
+)
+
+type SqliteDatabase struct {
+	*sql.DB
+}
+
+func sqliteInit(dbParams map[string]interface{}) (*SqliteDatabase, error) {
+
+	dbFilename, ok := dbParams["file"].(string)
+	if !ok {
```

Comment:
If conversion of the value fails we now return the error "err.conn.parse.sqlite.file.missing"